src/Persistence/json/jsonconfigstore.py

```diff
@@ -0,0 +1,25 @@
+import json
+from abc import ABC
+
+from Persistence.abstractconfigstore import ConfigStore
+
+
+class JsonConfigStore(ConfigStore, ABC):
```

Comment:
Does this assign 2 base classes?
If so, that seems a bit strange to me since ConfigStore already extends ABC?

Reply:
Nope, just one base class in practice (ConfigStore)

Reply:
But it is necessary to repeat the ABC here even though you already specified it in ConfigStore?